commands/climber/fullextend.py

```diff
@@ -0,0 +1,71 @@
+from commands2 import CommandBase
+from wpilib import RobotBase
+from subsystems.climbers.leftclimbersubsystem import LeftClimber
+from subsystems.climbers.rightclimbersubsystem import RightClimber
+import constants
+
+
```

Comment:
I had created something like that before but Luke asked for it to be both buttons held instead of a separate command. I guess in this case since we're just doing mid it doesn't matter so I can add it, or we can simply press both buttons